### DISCUSSION TITLE
Can you consider supporting multiple GPUs to participate in decoding?

### DISCUSSION THREAD

**Discussion by webclient163** [SuRGeoNix/Flyleaf#325] [Ideas]
When I need to play a lot of H264 videos, I find that one GPU (Nvidia GTX1050 Decoder) has reached 100%+ overload, but the other GPU (Inter UHD 630) is not involved in decoding. Is there a way to use a computer to decode with multiple GPUs?

**Comment by SuRGeoNix** [comment]
It is already supported. You can use Config.Video.GPUAdapter to specify a supported GPU which can be found in Engine.Video.GPUAdapters.  This way you can have for example two players with two different GPUs for decoding/rendering.
